README.md

```diff
@@ -1,6 +1,8 @@
 new-pykube
 =========
 
+[![PyPI version](https://badge.fury.io/py/new-pykube.svg)](https://badge.fury.io/py/new-pykube) [![Downloads](https://static.pepy.tech/badge/new-pykube/month)](https://pepy.tech/project/new-pykube) ![Python Test](https://github.com/caas-team/new-pykube/actions/workflows/python-tests.yml/badge.svg)
```

Comment:
I would change the link to link to a respective site, not the badge site. 
PyPI version to the pipy site of new-pykube
Downloads to the GitHub releases
And Python Test to the last workflow results (log output)

Reply:
ok, updated the links